webpack.config.js

```diff
@@ -6,6 +6,9 @@ const config = {
   output: {
     filename: 'webml-polyfill.js',
     path: path.resolve(__dirname, 'dist')
+  },
+  node: {
+    fs: 'empty'
```

Comment:
What's this for? loading worker? Probably to add a comment.

Reply:
It seems that even if I compile the wasm with `-s NO_FILESYSTEM=1` or `-s FILESYSTEM=0`, the resulting nn_ops.js still has reference to `fs` module. It will result in the error of
```
    ERROR in ./src/nn/wasm/nn_ops.js
    Module not found: Error: Can't resolve 'fs' in '/home/yueli/webshare/pinzhen/master/webml-polyfill/src/nn/wasm'
     @ ./src/nn/wasm/nn_ops.js 53:32-45
     @ ./node_modules/babel-loader/lib!./src/nn/wasm/PreparedModelWorker.js
```